tools/webots.py

```diff
@@ -178,9 +198,9 @@ def exec_push():
         sys.exit(exit_code)
 
 
-def run(sub_command, roles=None, role=None, **kwargs):
+def run(sub_command, roles=None, role=None, target="generic", **kwargs):
```

Comment:
IMO the default should be this. AMD and old intel users can use the flag lel
```suggestion
def run(sub_command, roles=None, role=None, target="g4dnxlarge", **kwargs):
```

Reply:
@JosephusPaye what do you reckon

Reply:
I think It should default to "generic" because we will likely only be running on aws for this robocup

Reply:
we're also only planning to use this webots build tool for this robocup?

Reply:
I believe the webots tool will still be useful after robocup.

Reply:
Set the default to `g4dnxlarge`, if people want to build generic they can do a normal build.

As this module stand it makes sense to have this target, if we want to change it in the furture, we can change the default then.

Reply:
Finneee